digiwf-libs/digiwf-testing/digiwf-unit-test/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>de.muenchen.oss.digiwf</groupId>
+        <artifactId>digiwf-testing</artifactId>
+        <version>1.4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>digiwf-unit-test</artifactId>
+    <description>Unit test lib by DigiWF</description>
```

Comment:
If I set this to packaging type pom, I couldn't use the `digiwf-unit-test` artifact as dependency. But now it creates an empty jar. Is there a better way to declare this module?

Reply:
You could add an additional `src/main/resources/.gitkeep` file to kill the warning. Eventually, some test utils can be put there.